src/IntercomContacts.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+namespace Intercom;
+
+use Http\Client\Exception;
+use stdClass;
+
+class IntercomContacts extends IntercomResource
+{
+    /**
+     * Creates a Contact.
+     *
+     * @see    https://developers.intercom.com/intercom-api-reference/reference#create-contact
+     * @param  array $options
+     * @return stdClass
+     * @throws Exception
+     */
+    public function create(array $options)
+    {
+        return $this->client->post("contacts", $options);
+    }
+
+    /**
+     * Updates a Contact.
+     *
+     * @see    https://developers.intercom.com/intercom-api-reference/reference#update-contact
+     * @param  array $options
+     * @return stdClass
+     * @throws Exception
+     */
+    public function update(array $options)
+    {
+        return $this->client->put("contacts", $options);
+    }
+
+    /**
+     * Lists Contacts.
+     *
+     * @see    https://developers.intercom.com/intercom-api-reference/reference#list-contacts
+     * @param  array $options
+     * @return stdClass
+     * @throws Exception
+     */
+    public function getContacts(array $options)
```

Comment:
Let's add a default here so we don't need to pass empty arrays when we want to make a full list request 😄 
```suggestion
    public function getContacts(array $options = [])
```